src/App.js

```diff
@@ -1,16 +1,41 @@
 import React from 'react';
 import './App.css';
 import chatMessages from './data/messages.json';
+// import ChatEntry from './components/ChatEntry';
+import ChatLog from './components/ChatLog';
+import {useState} from 'react'
 
 const App = () => {
+  
+  const[textMessage, setTextMessage] = useState(chatMessages)
+  const likedMessage = (id) => {
+  const isliked = textMessage.map((text) =>{
+    if (text.id === id){
+      return {...text, liked: !text.liked}
+    } else{
+      return text;
+    }
+  })
+    setTextMessage(isliked);
+  }
+
+  const totalLikes = (data)=> {
+    const newList = data.filter((ele)=>ele.liked===true)
+    return newList.length
+  }
+
+  const likedCount = totalLikes(textMessage)  
   return (
     <div id="App">
       <header>
-        <h1>Application title</h1>
+        <h1>Chat Between {textMessage[0].sender} and {textMessage[1].sender}</h1>
```

Comment:
What if textMessage[0] and textMessage[1] were both messages from the same person? How else could you find a way to get the names of the senders?